rastreamento_correios/README.md

```diff
@@ -0,0 +1,11 @@
+# O que é
+O "rastreamento_correios" é um scrypt feito em python com a intenção de rastrear um pedido qualquer utilizando técnicas de raspagem de dados aplicadas ao site dos correiros
+
+# Como rodar?
+Para executar este scrypt e obter um posicionamento sobre o seu pedido é simples, você precisa
+- Ter um pedido
+- - Geralmente quando você compra algo on-line este vem com um numero de rastreio ('SS123456789BR' é o exemplo de um!), ele possui 13 caracteres e podem ser encontrados no seu comprovante de postagem!
+- Informar o código de rastreio do seu pedido junto a linha de execução do seu scrypt
+- - ex `python rastreamento_correios SS123456789BR`
+
+Dessa forma você executará o script em questão e ele utilizará o código que você informou como parámetro de pesquisa!
```

Comment:
`parámetro` :arrow_right: `parâmetro`

e o de baixo não tá errado, mas uma sugestão de melhoria seria:
`código que você informou` :arrow_right: `código informado`

o pronome já tá em uso no início da frase, não precisa de repetição :wink: